api/serializers.py

```diff
@@ -783,7 +784,18 @@ class TaskListSerializer(serializers.Serializer):
     status = serializers.CharField(read_only=True)
     start_date = serializers.DateTimeField(read_only=True)
     finish_date = serializers.DateTimeField(read_only=True)
+    to_terminatable = serializers.SerializerMethodField()
```

Comment:
Why to_terminatable? Should be just terminatable, like upgradable

Reply:
Ok.

Reply:
Fixed